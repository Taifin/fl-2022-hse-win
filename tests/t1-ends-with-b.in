$start$ = $$S$$;
$S$ = ##a#$S$ | #b#;