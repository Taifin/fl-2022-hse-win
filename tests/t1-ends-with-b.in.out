LexToken(TERM,'$start$',1,0)
LexToken(BIND,'=',1,8)
